core/ui/EditTemplate/body/default.tid

```diff
@@ -0,0 +1,38 @@
+title: $:/core/ui/EditTemplate/body/default
+
+\define config-visibility-title()
+$:/config/EditorToolbarButtons/Visibility/$(currentTiddler)$
+\end
+
+\define importFileActions()
+<$action-popup $state=<<importState>> $coords="(0,0,0,0)" $floating="yes"/>
+\end
+
+<$set name="edit-preview-state" value={{{ [{$:/config/ShowEditPreview/PerTiddler}!match[yes]then[$:/state/showeditpreview]] :else[<qualify "$:/state/showeditpreview">] }}}>
+<$vars importTitle=<<qualify $:/ImportImage>> importState=<<qualify $:/state/ImportImage>> >
```

Comment:
We can combine the above two lines into a single $let widget now.

Reply:
I just lifted and shifted the current code to make the diffs easier to follow.

In terms of core usage of `<$let>` I wonder if this might be the time for a systematic review of the core templates.